index.js

```diff
@@ -0,0 +1,50 @@
+function validatePassword(valid) {
```

Comment:
Naming this `valid` is weird, it isn't always valid. It is alway a password, that'd be a better name.